app/model/Group.php

```
@@ -0,0 +1,238 @@
+        $group->update($updateData);
+
+        if (isset($data['users'])) {
+            $this->updateUsersInGroup($id, $data['users']);
+        }
+    }
+
+    /**
+     * @param int $id
+     * @return array|null
+     */
+    public function getGroupData($id)
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)